dependencies.list

```diff
@@ -1,2 +1,2 @@
-REALM_CORE_VERSION=2.8.1
-REALM_SYNC_VERSION=1.10.0
+REALM_CORE_VERSION=2.8.4
+REALM_SYNC_VERSION=2.0.0-rc9
```

Comment:
If we merge this to master, then the sync version should be kept as the current stable (`1.x.x`) so CI can test with the minor release i think?